search_indices/serializers/base.py

```diff
@@ -30,5 +30,10 @@ class Meta:
     def get_score(self, obj: Hit) -> int:
         return obj.meta.score
 
+    @property
+    def is_authenticated(self):
+        request = self.context.get("request")
+        return request and request.user.is_authenticated
```

Comment:
IMO
```suggestion
        return bool(request and request.user.is_authenticated)
```
would be a bit better because the name suggests the property will be a boolean, and the original version can return also something else, at least `None`.